easybuild/easyconfigs/n/NINJA/NINJA-0.97-cluster_only-GCC-10.2.0.eb

```diff
@@ -0,0 +1,29 @@
+# Author: Jasper Grimm (UoY)
+
+easyblock = 'MakeCp'
+
+name = 'NINJA'
```

Comment:
To avoid confusion: this was renamed to `TWL-NINJA` to avoid name clash with `Ninja`, cfr. #13412 and #13529